addons/binding/org.openhab.binding.harmonyhub/src/main/java/org/openhab/binding/harmonyhub/handler/HarmonyHubHandler.java

```diff
@@ -113,6 +113,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 } catch (NumberFormatException ignored) {
                     client.startActivityByName(command.toString());
                 }
+            } catch (IllegalArgumentException e) {
```

Comment:
In general I would say that the client should throw a more proper exception, then such a generic exception because now we might potentially catch others things, for which we can log no details without recompiling the binding, although the wrapped block is pretty small.

I would vote for a mwah (might not translate that well, but see it as an improvement with some resistance).

Reply:
I think the IAE is pretty clear to be thrown only for this cause, so I think that this catch-clause should be alright.

What is "mwah"? You indeed need to help me with the translation :-)

Reply:
I meant to explain it in between the brackets but failed with some autocorrect and typos. An approval with some resistance